lib/hope-tab-list.js

```diff
@@ -96,16 +101,13 @@ fxosComponent.register('hope-tab-list', {
   },
 
   getSnapHeight() {
-    return window.innerHeight -
-      PREVIEW_HEIGHT -
-      GUTTER_HEIGHT -
-      BAR_HEIGHT;
+    return window.innerHeight - TAB_HEIGHT;
   },
 
   getTabHeight() {
     return Math.max(
       TAB_HEIGHT,
-      this.getSnapHeight() / this.els.tabs.length
+      (this.getSnapHeight() + GUTTER_HEIGHT) / (this.els.tabs.length - 1)
     );
```

Comment:
I'd like a bit of JSDoc around this function to explain when/why the tab height is different.